c++/test/TestWriter.cc

```diff
@@ -2044,103 +2044,5 @@ namespace orc {
     testSuppressPresentStream(CompressionKind_SNAPPY);
   }
 
-  TEST(WriterTest, testSeekWithNestedType) {
```

Comment:
Why is it removed?

Reply:
It just move into TestReadIntent.testSeekOverEmptyPresentStream in the TestReader.cc file.